barretenberg/cpp/src/barretenberg/vm2/simulation/gadgets/tx_execution.cpp

```diff
@@ -71,14 +71,14 @@ TxExecutionResult TxExecution::simulate(const Tx& tx)
         .phase_lengths = PhaseLengths::from_tx(tx), // extract lengths of each phase at start
     });
 
-    info("Simulating tx ",
-         tx.hash,
-         " with ",
-         tx.setupEnqueuedCalls.size(),
-         " setup enqueued calls, ",
-         tx.appLogicEnqueuedCalls.size(),
-         " app logic enqueued calls, and ",
-         tx.teardownEnqueuedCall ? "1 teardown enqueued call" : "no teardown enqueued call");
+    vinfo("Simulating tx ",
```

Comment:
This gets noisy during fuzzing, alternatively could wrap with a `#ifndef` block

Reply:
LG as vinfo